quiz.css

```diff
@@ -0,0 +1,76 @@
+body {
+  margin-left: 1rem;
+}
+body .product {
+  margin: 5px;
+  display: inline-block;
+}
+
+.product {
+  border: 1px solid lightblue;
+  border-radius: 5px;
+  margin:.75em;
+  padding: 10px;
+  font-family: 'Lora', serif;
+  margin-top: 5em;
+}
+
+.product h1 {
+  border-bottom: 1px dotted lightgrey;
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  padding-bottom: 15px;
+}
+
+.product-image{
+    width:100%; 
+    display: block;
+    height: auto;
+    margin: auto;
+}
+
+
+.description {
+  text-align: justify;
+}
+
+.pricing p {
+  font-family: 'Lora', serif;
+  font-weight: 700;
+}
+
+.availability {
+  background: grey;
+  color:yellow;
+}
+
+ .product footer: {
```

Comment:
You have a misplaced colon. Also, you are being overly specific, simply use `footer`